modules/katello_repository.py

```diff
@@ -73,6 +73,13 @@
             - immediate
             - on_demand
         required: false
+    state:
+        description:
+            - State of the Repository
+        default: true
+        choices:
+        - present
+        - absent
```

Comment:
Do you mind adding 'present_with_defaults' here and inline 148?
Cements naildown should make it _just work_.